src/Tribe/Google/Maps_API_Key.php

```diff
@@ -50,15 +50,12 @@ public static function instance() {
 	public function filter_tribe_addons_tab_fields( array $addon_fields ) {
 
 		$tooltip = sprintf(
-			__( '<p><strong>You are using a custom Google Maps API key.</strong></p><br><p>%1$s to learn more about using it with The Events Calendar.</p>', 'the-events-calendar' ),
+			__( '<p><strong>You are using a custom Google Maps API key.</strong></p><p>%1$s to learn more about using it with The Events Calendar.</p>', 'the-events-calendar' ),
```

Comment:
It is a bit of a pain as it was already here, but lets separate out the html tags from the translation. 

Something like this:

'<p><strong>%1$s</strong></p><p>%2$s%3$s</p>'
__( 'You are using a custom Google Maps API key.', 'tec' ),
__( 'to learn more about using it with The Events Calendar.', 'tec' ),

Reply:
fixed!